### DISCUSSION TITLE
How to get in contact for demo?

### DISCUSSION THREAD

**Discussion by CGRemakes** [pimcore/pimcore#17951] [Q&A / Support]
I'm reaching out here, because I'm not sure where else to go.  I used Pimcore quite extensively about 4 years ago while with a previous employer.  I ended up leaving that company, and went a few years without using it.

I recently joined a company where I feel Pimcore would be a good fit, so I reached out to get in contact with a rep that might be able to help us navigate this possibility.  I used both the standard email form and left messages using phone numbers found here: https://pimcore.com/en/contact-us.  This was several weeks ago, and after multiple attempts, have not heard back from anyone.

This lack of response has made me question whether Pimcore would be a good option, especially if we go the PaaS route.  I personally am sold on the product and the direction of Pimcore, but require better support if I'm going to recommend it to my current company.

Please let me know if there are better avenues to get in contact with someone, or what my next steps should be.

**Comment by fashxp** [comment]
Hi,
I'm sorry to hear that. Could be please retry once more our contact form and also refer to that post? 
I notified my colleagues as well...

Cheers and happy new year!!

BR
Christian

**Comment by drietsch** [comment]
Hello CGRemakes,

Dietz here, CEO of Pimcore. First of all, I want to sincerely apologize for the lack of response. That’s definitely not how we want to operate, and I appreciate you bringing this to my attention. I’ll personally ensure this gets addressed right away.

To get things moving as quickly as possible, feel free to contact me directly at dietmar.rietsch@pimcore.com, and I’ll make sure you’re connected with the right team to explore Pimcore for your company.

Thanks again for your patience and for believing in Pimcore. Let’s make this happen! Wishing you a fantastic start to the new year.

Best regards,
Dietz

**Reply by CGRemakes** [reply]
Thank you so much for your concerns and efforts.  I've followed up in an email to you.
